modules/constructor/constructor.services.js

```diff
@@ -22,20 +24,34 @@ class ConstructorService {
     throw new Error(error);
   }
 
-  async addConstructorElement(data, model, error) {
-    if (await this.checkConstructorElementExist(data, model)) {
+  async addConstructorElement({ constructorElement, upload }, model, error) {
+    if (upload) {
+      constructorElement.image = await uploadLargeImage(upload);
```

Comment:
Maybe you need `thumbnail` image for avatar

Reply:
we don`t need Avatar on the front-end, but now i use 
![image](https://user-images.githubusercontent.com/46137635/107724563-d1085d00-6cec-11eb-9528-d55bb719dd8d.png)
uploadSmallImage() on front-end for image resize to 
![image](https://user-images.githubusercontent.com/46137635/107724631-fd23de00-6cec-11eb-8f59-3a1531b38368.png)
because 768px resolution